src/main/java/crazypants/enderio/machine/spawner/TilePoweredSpawner.java

```diff
@@ -211,10 +268,10 @@ protected IPoweredTask createTask(IMachineRecipe nextRecipe, float chance) {
     } else {
       ticksDelay = TilePoweredSpawner.MAX_SPAWN_DELAY_BASE - ((TilePoweredSpawner.MAX_SPAWN_DELAY_BASE - TilePoweredSpawner.MIN_SPAWN_DELAY_BASE) / 2);
     }
-    if(getCapacitorType().ordinal() == 1) {
-      ticksDelay /= 2;
-    } else if(getCapacitorType().ordinal() == 2) {
-      ticksDelay /= 4;
+    if(getCapacitor().getTier() < 3) {
+      ticksDelay /= Math.pow( 2 , (getCapacitor().getTier() - 1 )) ;
```

Comment:
That looks suspicious. Probably "-1" is redundant

Reply:
a spawn without capacitor is with a tier 1 capacitor, so speed is 2^(1-1)=1

Reply:
yes, so the divisors now are 1, 2, and 8, unlike before

Reply:
(Also I hope Java compiler is smart enough to replace pow() with bit shift here)

Reply:
Still I think `ticksDelay >>= Math.min(3, getCapacitor().getTier() - 1);` would do the same thing cleaner